website/src/css/custom.css

```diff
@@ -4,43 +4,34 @@
  * work well for content-centric websites.
  */
 
-/* You can override the default Infima variables here. */
-:root {
-  --ifm-color-primary-lightest: #57e1a7;
-  --ifm-color-primary-lighter: #38dc97;
-  --ifm-color-primary-light: #2eda91;
-  --ifm-color-primary: #24cc85;
-  --ifm-color-primary-dark: #20b878;
-  --ifm-color-primary-darker: #1fad71;
-  --ifm-color-primary-darkest: #198f5d;
-}
+/* For readability concerns, you should choose a lighter palette in dark mode. */
+[data-theme="dark"],
+:root[data-theme="dark"],
+html[data-theme="dark"] {
+  /* lazily setting all colors to be the same, can't see the shades being used anywhere rn*/
```

Comment:
this is where the shades came from: https://docusaurus.io/docs/styling-layout#styling-your-site-with-infima

Also not sure how they are used.

Reply:
Added in the colors in case they are useful